packages/nodejs-lib/src/validation/ajv/getAjv.ts

```diff
@@ -131,5 +135,109 @@ function addCustomAjvFormats(ajv: Ajv): Ajv {
           return n >= -14 && n <= 14 && Number.isInteger(n)
         },
       })
+      .addFormat('isoDate', {
```

Comment:
Proposal to make it follow the TS interface: `IsoDate`.

Same for literal all others.

So, formats become PascalCase